README.md

```diff
@@ -8,14 +8,13 @@
 ## Setup
 * Clone the repo
 * Install dependencies `npm install`
-* Update `*.conf.js` files inside the `conf/` directory with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings)
+* Update `intern.json` with your [BrowserStack Username and Access Key](https://www.browserstack.com/accounts/settings)
 
 ## Running your tests
-* To run single test, run `npm run single`
-* To run local tests, run `npm run local`
-* To run parallel tests, run `npm run parallel`
+* To run sample test, run `npm run sample-test`
```

Comment:
Let's keep the scripts and conf.js files consistent with docs and then only rename and modify the files.

Reply:
Also, let's keep it consistent across all branches.

Reply:
This will be finally updated in the master branch along with the docs changes.